extern/jsoncpp/src/lib_json/json_writer.cpp

```diff
@@ -111,7 +111,7 @@ std::string valueToString( bool value )
 std::string valueToQuotedString( const char *value )
 {
    // Not sure how to handle unicode...
-   if (strpbrk(value, "\"\\\b\f\n\r\t") == nullptr && !containsControlCharacter( value ))
```

Comment:
This change was done in an automated manner in https://github.com/stepmania/stepmania/commit/ba59dd1656ebb275694d6df443b8c0490e260ba2

Obviously this is equivalent.